docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -116,33 +122,39 @@ Define the data you want GX Cloud to access within PostgreSQL.
 
 2. Click the **New Data Source** tab and then select **PostgreSQL**.
 
-3. Enter a meaningful name for the Data Asset in the **Data Source name** field.
+3. Select one of the following options:
 
-4. Enter a connection string in the **Connection string** field. The connection string format is `postgresql+psycopg2//YourUserName:YourPassword@YourHostname:5432/YourDatabaseName`. 
+    - If you're connecting to an org-hosted PostgreSQL Data Asset for the first time, copy the code and see [Connect GX Cloud to PostgreSQL](../connect/connect_postgresql.md).
 
-5. Optional. Select **Test connection** to test the Data Source connection. Testing the connection to the Data Source is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
+    - If you're testing GX Cloud features and functionality in a self-hosted environment, click **I have created a GX Cloud user with valid permissions** and then click **Continue**.
 
-6. Click **Continue**.
+4. Enter a meaningful name for the Data Asset in the **Data Source name** field.
```

Comment:
I don't think this is supposed to be a "meaningful name for the Data Asset."  Shouldn't this be a meaningful name for the Data Source, and the Data Asset gets named later?

Reply:
@Rachel-Reverie, this issue has been addressed. The UI has been changing frequently, so this was missed.